tests/c/instruction_checks/get_branch_vram_generic.c

```diff
@@ -0,0 +1,17 @@
+/* SPDX-FileCopyrightText: © 2023 Decompollaborate */
+/* SPDX-License-Identifier: MIT */
+
+#include "rabbitizer.h"
+
+#include <stdlib.h>
+
+int main() {
+    RabbitizerInstruction inst;
+    RabbitizerInstruction_init(&inst, 0x14600001, 0x80000000); // bnez        $v1, . + 4 + (0x1 << 2)
+    RabbitizerInstruction_processUniqueId(&inst);
+    if (RabbitizerInstruction_getBranchVramGeneric(&inst) < 0) {
```

Comment:
Since this function returns an unsigned type then this check will always be false. A bit surprising if the compiler doesn't complain about this

Reply:
I had the same thought, but the compiler didn't warn, and I thought it might be good to have some test. I'll remove it if you don't think it's useful.

Reply:
Changing the compiler to gcc makes it complain
```
$ make CC=gcc
# ...
: warning: comparison of unsigned expression in ‘< 0’ is always false [-Wtype-limits]
```

Seems like clang doesn't enable `-Wtype-limits` as part of `-Wextra` like gcc does (https://reviews.llvm.org/D142826), which is a bit unfortunate